docs/.vuepress/tools/jsdoc-convert/jsdoc.js

```diff
@@ -239,19 +239,18 @@ const fixLinks = text => text
       if (target.includes('://')) { // e.g https://handsontable.com/blog
         return all;
       }
-      let fixedAnchor = anchor
-        .toLowerCase();
+      let fixedAnchor = anchor.toLowerCase();
 
       // e.g. #Options+autoColumnSize or #getData
       if (!target) {
         if (!fixedAnchor.includes('+')) {
           return `[${label}](${hash}${fixedAnchor})`;
         } else {
-          const splitted = fixedAnchor.split('+');
+          const splitted = anchor.split('+');
```

Comment:
Nitpick, but this should probably be named `split`, not `splitted` (or preferably something more descriptive?)

Reply:
done d02b7c1801e81b3c65f6644fe64f3be226181d0c